backend/src/apis/ownership.rs

```diff
@@ -0,0 +1,37 @@
+use crate::contracts::{generated::summa_contract::AddressOwnershipProof, signer::SummaSigner};
+use ethers::types::Address;
+use std::{error::Error, result::Result};
+
+pub struct AddressOwnership {
+    address_ownership_proofs: Vec<AddressOwnershipProof>,
+    signer: SummaSigner,
+}
+
+impl AddressOwnership {
+    pub fn new(
+        main_signer_key: &str,
+        chain_id: u64,
+        rpc_url: &str,
+        address: Address,
```

Comment:
name it `summa_sc_address`

Reply:
Updated comment at [ce908f0](https://github.com/summa-dev/summa-solvency/pull/152/commits/ce908f0a7323153aea8f3f959f16e69766a661d7)